README.md

```diff
@@ -87,6 +100,31 @@ cargo build --workspace
 cargo test --workspace
 ```
 
+### Using Podman instead of Docker
+
+If you want to use Podman you have to [enable Podman in Visual Studio
+Code][vscode-podman] and update the `.devcontainer/devcontainer.json` file
+with the following additions:
+
+  [vscode-podman]: https://code.visualstudio.com/remote/advancedcontainers/docker-options#_podman
+
+```Dockerfile
+  "runArgs": [
+    "--cap-add=SYS_PTRACE",
+    "--security-opt",
+    "seccomp=unconfined",
+    "--userns=keep-id"
+  ],
+
+  ...
+
+  "workspaceMount": "source=${localWorkspaceFolder},target=/workspace,type=bind,Z",
+  "workspaceFolder": "/workspace",
+  "containerUser": "vscode"
+```
```

Comment:
> The snippet should just help getting the right changes, maybe splitting them into separate chunks?

How about just putting it in [JSONC](https://changelog.com/news/jsonc-is-a-superset-of-json-which-supports-comments-6LwR) format (and which is what this is instead of `Dockerfile`) where you use `/* ... */` to mark _other stuff_?

```suggestion
```jsonc
{
  /* ... */
  "runArgs": [
    "--cap-add=SYS_PTRACE",
    "--security-opt",
    "seccomp=unconfined",
    "--userns=keep-id"
  ],

  /* ... */
    
  "workspaceMount": "source=${localWorkspaceFolder},target=/workspace,type=bind,Z",
  "workspaceFolder": "/workspace",
  "containerUser": "vscode",
  /* ... */
}
```
```